druid/src/core.rs

```diff
@@ -721,7 +747,7 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
         self.old_data = Some(data.clone());
         self.env = Some(env.clone());
 
-        ctx.base_state.merge_up(&self.state)
+        ctx.base_state.merge_up(&self.state);
```

Comment:
nitty rust thing, but I don't feel a semicolon is necessary here? we're returning nothing, this last method returns nothing, we can just let it evaluate? (not a blocker just an observation)

Reply:
I'm guessing the upside of the semicolon is that if `merge_up` starts returning a value then without a semicolon this will be a compile error.